site/content/entry/improvements-to-hardware-renderer.md

```diff
@@ -0,0 +1,95 @@
++++
+date = "2018-02-28T17:00:00-04:00"
+title = "Speeding up Citra: Improvements to the Hardware Renderer"
+tags = [ "feature-update" ]
+author = "jroweboy"
+forum = 
++++
+
+## Cut to the chase, how fast is this?
+
+Very. Test results across various computers show that it averages out to be a **2x speed boost for almost everyone.**
```

Comment:
Using average and almost in the same sentence makes the statement weak. I would take out "for almost everyone" considering there's a wide variance.